PlayCover/Model/PlayApp.swift

```diff
@@ -107,6 +107,19 @@ class PlayApp: BaseApp {
         }
     }
 
+    var aliasDirectory: URL {
```

Comment:
Instead of making this a computed property, it can be a static constant, which could be easier to read and use.